sample/test.py

```diff
@@ -1,15 +1,22 @@
 from sample.fraction import Fraction
 
+
+def assert_equals(expect, actual):
```

Comment:
assert_eqaul 이 더 나은 것 같아요.

```suggestion
def assert_equal(expect, actual):
```